README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/paypal/NNAnalytics.svg?branch=master)](https://travis-ci.com/paypal/NNAnalytics)
```

Comment:
I think `https://travis-ci.org/paypal/NNAnalytics.svg?branch=master` -> `https://travis-ci.com/paypal/NNAnalytics.svg?branch=master`, right?

Reply:
needs travis-ci.com in both places.